# Bio 

My name is Ashlynn Steeves, I am a fourth year Biomedical Engineering student from the University of Victoria. I am working with the CFIA from September to December 2019 to complete my fourth and final co-op placement. During this placement I will be participating in the development of predictive genomics software, and the analyses of genome sequence data in support of the national brucellosis program.

# Background 

Brucellosis is a disease caused by several species of the Brucella bacterium. It is chronic, contagious and a zoonotic disease, meaning it can spread from animals to humans. There are various types of brucellosis, including:  

   - Brucella abortus, which primarily affects cattle, bison and elk
   - Brucella suis, which primarily affects swine
   - Brucella melitensis, which primarily affects goats and sheep

In Canada, brucellosis is a disease that is readily controlled through federal surveillance and eradication programs. However, the threat posed by brucellosis is still fairly wide spread. Its presence in bison in northern Alberta continues to pose significant risk to expansion of livestock (cattle and bison) operations and re-establishment of wood bison in the area. Brucellosis is also a concern in domestic cattle as an outbreak can result in significant economic losses. Furthermore, it is a potential human health risk, as humans can become infected by all types of brucellosis, resulting in a disease called undulant fever.

Previous research projects have enabled implementation of standard PCR tools such as the Bruce ladder technique for identification of many brucella species. However, such techniques are quite technically cumbersome and time-consuming and would benefit from replacement with more rapid, quantifiable and sensitive methods, such as real-time PCRs. Development of such tools is however stymied by a lack of suitable markers for species. 

In this study, a large-scale comparative whole genome sequence data analysis will seek unique markers for identification of brucellosis at the species and biovar levels.This information will assist in the development of new more efficient molecular tools for isolate identification. Such tools will further improve CFIAâ€™s surveillance and import control program.


# Methods

# Expected results 

As a result of the work to be completed this term we expect to obtain a series of markers that can be used to identify different species of brucella. Such markers could be the persence of absense of a genomic region or single nucleotide varriants in shared genes. Ideally these markers would then be implemented in downstram tests such as PCR diagnostics and metagenomics. 

# Refrences 
 - http://www.inspection.gc.ca/animals/terrestrial-animals/diseases/reportable/brucellosis/fact-sheet/eng/1305673222206/1305673334337
 - https://open.alberta.ca/dataset/a4203bfc-914c-4a10-b699-47f84d8894a0/resource/31538d8b-be69-415d-8d0d-27793a20651a/download/brucellosis-mar-2004.pdf
 - GRDI Brucellosis Research proposal 